src/Components/Blazor/Build/src/targets/Blazor.MonoRuntime.targets

```diff
@@ -157,31 +210,30 @@
     </ItemGroup>
   </Target>
 
-  <UsingTask TaskName="BlazorILLink" AssemblyFile="$(BlazorTasksPath)" />
   <UsingTask TaskName="GenerateTypeGranularityLinkingConfig" AssemblyFile="$(BlazorTasksPath)" />
+  <Target Name="_GenerateTypeGranularLinkerDescriptor"
```

Comment:
@SteveSandersonMS this is now incremental:

![image](https://user-images.githubusercontent.com/174281/72177714-28bff080-3396-11ea-8388-81a21a9de74c.png)

My guess is that for `BlazorCreateRootDescriptorFile`, the inputs doesn't need to be `@(IntermediateAssembly)` - we don't need to recreate the file, if the contents of the file changes,only it's name which is almost never true, but I guess we can figure that out later.